src/main/java/eu/europeana/metis/sandbox/config/IndexingConfig.java

```diff
@@ -45,6 +45,9 @@
   @Value("${sandbox.preview.mongo.db}")
   private String mongoPreviewDb;
 
+  @Value("${sandbox.preview.mongo.application-name}")
```

Comment:
Okay it looks good now, but I think we need to have it get default value of null like the other fields otherwise Spring might interpret it as empty String, did you check that?
(Same for the publish one)

Reply:
When spring does not find a match for the property it will throw an exception with a message similar to:
```java
nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'test.myprop.here' in value "${test.myprop.here}"
```
Unless there is a property with that name but with an empty value, in that case spring will assign an empty string value

Reply:
Thank you! That is correct, I have just tested as well. I will add the default null values.